packages/react-native/template/README.md

```diff
@@ -0,0 +1,78 @@
+This is a [**React Native**](https://reactnative.dev) project bootstrapped with [`npx react-native@latest init`](https://github.com/react-native-community/cli/tree/main#creating-a-new-react-native-project).
+
+# Getting Started
+
+>**Note:** Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till [Creating a new application](https://reactnative.dev/docs/environment-setup#:~:text=Creating%20a%20new%20application) step, before proceeding.
+
+## Step 1: Start the Metro Server
+
+First, you will need to start Metro, the JavaScript _bundler_ that ships _with_ React Native. Metro "takes in an entry file and various options, and returns a single JavaScript file that includes all your code and its dependencies."—[Metro Docs](https://facebook.github.io/metro/docs/concepts)
+
+To start Metro, run `yarn start` _inside your React Native project folder_:
+
+```bash
+yarn start
+```
+
+Doing so, will starts the **Metro Bundler**.
+
+>**Note:** You can use `npx` instead of `yarn` when running React Native commands inside an existing project. Just replace `yarn` with `npx` and follow the commands (syntax) provided in `package.json` **`scripts`**.
```

Comment:
I'd remove this for the same reason mentioned in the other comment, we should steer away from npx aside from init
```suggestion
```

Reply:
Ok ok 😅